packages/client/src/views/SysAdmin/Team/user/UserList.tsx

```diff
@@ -198,53 +184,24 @@ const ChangeButton = styled(LinkButton)`
   }
 `
 
-const NameRoleTypeContainer = styled.div`
-  margin-right: auto;
-  display: flex;
-  flex-direction: column;
-`
-
-const StatusMenu = styled.div`
+const StatusMenuContainer = styled.div`
   display: flex;
   align-items: center;
 `
 
-const Name = styled(LinkButton)`
-  align-self: flex-start;
-  text-align: left;
+const Value = styled.div`
```

Comment:
```suggestion
const Value = styled.span`
```
This should fix the alignment issues